src/pages/wallet/wallet-import-manual/wallet-import-manual.ts

```diff
@@ -140,9 +172,7 @@ export class WalletManualImportPage extends BaseWalletImport  {
     if (Math.abs(lastWordLastPassphrase.length - lastWordPassphrase.length) === 1 && lastWordPassphrase.length > 1 &&
         (lastWordLastPassphrase.indexOf(lastWordPassphrase) !== -1 || lastWordPassphrase.indexOf(lastWordLastPassphrase) !== -1 )) {
       // we just want one letter to be different - only "manual" typing, don't suggest on copy/paste stuff
-      const bip39 = require('bip39');
-      const englishWordlist = bip39.wordlists.english;
-      this.wordSuggestions = englishWordlist.filter( word => word.indexOf(lastWordPassphrase) === 0 );
+      this.wordSuggestions = this.wordlist.filter( word => word.indexOf(lastWordPassphrase) === 0 );
```

Comment:
A little pedantic but would you mind removing the spacing in the brackets please, just to keep it in the style of everywhere else? thanks!

Reply:
No problem, changed it 😉